_posts/faq.md

```diff
@@ -24,7 +24,16 @@ Abbiamo inoltre altri profili social, che puoi trovare in fondo ad ogni pagina,
 
 Uno degli aspetti caratterizzanti del JUG Milano sono gli incontri dove viene presentato e dibatutto un tema a scelta di interesse per il gruppo; alla <a href="meetings">sezione meetings</a> è disponibile un archivio degli incontri precedenti e relativo materiale.
 
-Se vuoi proporti come speaker per un incontro puoi candidari in Mailing-list oppure contattando i coordinatori del JUG Milano.
+Se vuoi proporti come speaker per un incontro puoi candidari in Mailing-list (trovi
+<a href="https://groups.google.com/g/jugmilano/c/PCkO-Z7X_vU/m/2JYCr1iAAgAJ" target="_blank">alcuni</a>
+<a href="https://groups.google.com/g/jugmilano/c/_D7m1a7pI5g/m/_b3g226BAgAJ" target="_blank">esempi</a>
+<a href="https://groups.google.com/g/jugmilano/c/MPLx_Iqfcuk/m/JVrL2y41BgAJ" target="_blank">qui</a>
+), oppure contattando i coordinatori del JUG Milano.
+
+Non saranno accettate proposte che manifestino promozione diretta di prodotti o servizi di natura unicamente commerciale o di marketing.
```

Comment:
Questa frase mi sembra poco chiara, il concetto è spiegato (ripetuto) meglio nei paragrafi successivi.
Magari si può riassumere in un testo più sintetico, io partirei dalle riga 35 che mi sembra quella che esprime meglio le finalità del gruppo.

Reply:
@fvasco io invecce trovo sia molto chiara e l'avverbio `unicamente` pone la distinzione che permette di fare talk anche basati su prodotti ma non devono esserlo esclusivamente: insomma, va bene se parli del tuo prodotto, ma solo se lo fai nel contesto di un talk dove presenti il contesto, le sfide e il tuo risultato. Di modo che, anche se non usiamo il tuo prodotto, comunque abbiamo imparato qualcosa sul problema che quel prodotto vuole risolvere.

Reply:
@andreaiacono quello che hai scritto tu mi sembra più chiaro di quanto proposto: hai esplicitato un concetto che era implicito.
Il rigo 35 affronta già questo problema e ne spiega meglio i termini, per questo propongo di trattarlo lì in modo più organico.

Provo a fare un merge:

> Vogliamo creare un ambiente in cui l'apprendimento e la condivisione siano al centro dell'attenzione, valorizzando le idee e l'arricchimento personale. Le proposte di promozione di prodotti o servizi unicamente commerciali saranno rifiutate.